alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/task/StartupScheduledTask.java

```diff
@@ -47,7 +48,7 @@ public void checkTaskEnabled() {
     public void startTask() {
         checkTaskEnabled();
         String taskName = getTaskName();
-        if (!getEnabled()) {
+        if (BooleanUtils.isFalse(getEnabled())) {
```

Comment:
Would returning a `boolean` instead of a `Boolean` resolve this? I'd rather not use `BooleanUtils` everywhere.

Reply:
Good catch

Reply:
I believe SonarCloud won't complain if you do the following:
`boolean enabled = getEnabled();
if(!enabled) {`

Although you still have the same underlying problem with Null.  We should favor the primitive type, but the Boolean Object might be needed for the databased entity objects.  I'd have to refresh my memory on that.